proposed/container-meta.md

```diff
@@ -239,7 +239,16 @@ While `ContainerInterface` only defines one mandatory parameter in `get()`, it i
 existing containers that have additional optional parameters. PHP allows an implementation to offer more parameters
 as long as they are optional, because the implementation *does* satisfy the interface.
 
-This issue has been discussed in [container-interop's issue #6](https://github.com/container-interop/container-interop/issues/6).
+Difference with container-interop: [The container-interop spec](https://github.com/container-interop/container-interop/blob/master/docs/ContainerInterface.md) stated that:
+
+> While `ContainerInterface` only defines one mandatory parameter in `get()`, implementations MAY accept additional optional parameters.
+
+This sentence was removed from PSR-11 because:
+
+- It is something that stems from OO principles in PHP, so this is not directly related to PSR-11
+- We do not want to encourage implementors to add additional parameters as we recommend coding against the interface and not the implementation
+
+Yet, some implementations have extra optional parameters; that's technically legal. Such implementations are compatible with PSR-11. [[6]](#link_get_optional_parameters)
```

Comment:
@Crell , I agree when you say:

>  Most other spec metadocs have had a specific section dedicated to external links, such as key polls or such, but not in the prose of the metadoc itself. Having the links interspersed implies "you really need to go read these to understand this", aka, "this metadoc is incomplete."

I also agree with @stefanotorresi when he says:

> historically there have been a few instances were outsiders posted doubts, questions or critiques on the ML, only to recieve the response "this has already been dicussed in link", so keeping track of such references on the metadoc should be welcomed.

I've changed this commit to synthesize your point of views:

I'm simply putting a small reference at the end of the chapter that points to the "links" section at the end of the document.

That way, the meta-document remains readable by itself (as requested by @Crell), and yet, I'm adding more links at the end of the document so that people who want to deep dive in the discussions can do it (as requested by @stefanotorresi )

Are you ok with this approach? If yes, I'll submit another PR that moves all other links of the meta doc to the links section at the bottom of the doc.

On a side note, I also removed the reference to the previous version of the spec, but kept the reference to container-interop (as this is something I believe is important to track since PSR-11 will deprecate container-interop).